### PR TITLE
feat(dsg): add DSG version to logs page

### LINKED ISSUE #4468: Add DSG version to the build logs
### Feature description

Add DSG version to the build logs

### Use case
In order to be sure which DSG version is running, it would be helpful to add a log with the DSSG version in the build page

_No response_

### Are you willing to submit PR?

_No response_

### PR DESCRIPTION
**Pull request by abrl91**
Close #4468 

## PR Details

add DSG vestion to logs page

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

### CODE REVIEW

File: packages/amplication-data-service-generator/src/server/create-server.ts [hunk 1]
```diff
@@ -43,6 +43,13 @@ async function createServerInternal(
     logger,
   } = DsgContext.getInstance;
 
+  const dsgVersion = (await import("../../package.json")).version;
+  logger.info(`running DSG version ${dsgVersion}`);
+  await createLog({
+    level: "info",
+    message: `running DSG version ${dsgVersion}`,
```
Comment (tupe12334) [inline packages/amplication-data-service-generator/src/server/create-server.ts:50]:
```suggestion
    message: `Running the DSG in version ${dsgVersion}`,
```

File: packages/amplication-data-service-generator/project.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
       ],
       "outputs": ["{projectRoot}/node_modules"],
       "options": {
-        "commands": ["npm ci"],
+        "commands": ["npm ci", "ts-node ./scripts/update-version.ts"],
```
Comment (tupe12334) [inline packages/amplication-data-service-generator/project.json:39]: I suggest adding this as a separate target to the project.json and call it as dependence. What do think?

File: packages/amplication-data-service-generator/src/server/create-server.ts (hunk 1 above)
Comment (abrl91) [reply packages/amplication-data-service-generator/src/server/create-server.ts:50]: I will change it to `Running DSG Version: ${dsgVersion}`

File: packages/amplication-data-service-generator/project.json (hunk 2 above)
Comment (abrl91) [reply packages/amplication-data-service-generator/project.json:39]: I don't think it is necessary. What are the benefits? Do we need this command elsewhere?

Comment (tupe12334) [reply packages/amplication-data-service-generator/project.json:39]: I think it's important to separate commands for there proper use; in that case, I will add `update-version` script that runs prebuild, preserve and prepublish.

**Comment by yuval-hazaz** [comment]
@abrl91 are we missing anything to move forward with this one?

**Comment by abrl91** [comment]
> @abrl91 are we missing anything to move forward with this one?

@arielweinberger can you review, please? I remember you said that you want to implement it in a different way (not using the package.json)

**Comment by arielweinberger** [comment]
I don't want to block the efforts so I'm okay with this for now.

**Review by arielweinberger — APPROVED** [review]

**Pull request merged**